03-lection1/02-accordion/accordion.css

```diff
@@ -1 +1,51 @@
 /* Стилизация аккордеона */
+
+summary:focus {
```

Comment:
Вижу у `summary` класс `accordion__summary`, можно было так)
```suggestion
accordion__summary:focus  {
```

Reply:
спасибо, поправил!